Create ui-c8y-1021-22-4-fix-app-switcher-alignment-in-mobile-7865-graft-release-y-2025.md (#2760)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-4-fix-app-switcher-alignment-in-mobile-7865-graft-release-y-2025.md

---
date: ""

> title: fix-app-switcher-alignment-in-mobile (#7865) [GRAFT][release/y2025] (#7881)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed alignment of app switcher in mobile view
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61682
version: 1021.22.4
---

> fix-app-switcher-alignment-in-mobile (#7865) [GRAFT][release/y2025] (#7881)

Reviewer: ```suggestion
The app switcher in the mobile view of Cumulocity was previously not aligned correctly. This issue has now been resolved. Users will now see the app switcher properly aligned in the center when using Cumulocity on mobile devices, improving the overall usability and visual consistency of the platform.
```

> fix-app-switcher-alignment-in-mobile (#7865) [GRAFT][release/y2025] (#7881)

Author: The Claude AI suggested to use the following body:
```suggestion
The app switcher in the mobile view of the Cumulocity user interface was previously not aligned correctly, causing a poor user experience. This issue has now been resolved by adjusting the alignment of the app switcher. Users will now see the app switcher properly aligned in the center when using Cumulocity on mobile devices, improving the overall usability and visual consistency of the platform.
```

Reviewer 2: maybe shorter? 
The app switcher in the mobile view was previously not aligned correctly. This issue has now been resolved, users will now see the app switcher properly aligned in the center when using Cumulocity on mobile devices, improving the overall usability and visual consistency of the platform.
